public/assets-new-ui/styles.css

```diff
@@ -0,0 +1,260 @@
+}
+
+/*= HEADER */
+
+header[role="banner"] {
+  background-color: #e9eff1;
+  border-bottom: 1px solid #dcdcdc;
+}
+
+.glogo {
+  width: 69px;
+  height: 69px;
+}
+
+/*= FOOTER */
+
+footer[role="contentinfo"] {
+  background: #333;
+  color: #999;
+}
+
+/* Support navigation */
+.support ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.support a {
+  color: white;
+  text-decoration: none;
+}
+
+/*= CONTENT */
+.content {
+  background: white;
+  padding: 0 10px;
+}
+
+.contribution-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form__field {
+  align-items: stretch;
+  display: flex;
+  flex-direction: column;
+}
+
+/*= BREAKPOINTS */
+
+@media (min-width: 720px) {
+  .content {
+    border-color: #dcdcdc;
+    border-style: none solid;
+    border-width: 0 1px;
+  }
+}
+
+@supports(display: grid) {
+  body {
+    display: grid;
+    grid-template-rows: [header-t] 75px [header-b] 1fr [footer-t] auto [footer-b];
+  }
+  
+  header[role="banner"] {
+    grid-row: header-t / header-b;
+  }
+  
+  footer[role="contentinfo"] {
+    grid-row: footer-t / footer-b;
+  }
+  
+  .content {
+    grid-row: header-b / footer-t;  
+  }
+    
+  @media (min-width: 720px) {
+    body,
+    header[role="banner"],
+    footer[role="contentinfo"] {
+      grid-column-gap: 20px;
+      grid-template-columns: 1fr repeat(8, 70px) 1fr;  
+    }
+    
+    header[role="banner"],
+    footer[role="contentinfo"] {
+      display: grid;
+      grid-column: span 10;
+    }
+    
+    .content {
+      margin: 0 -11px;
+        
+      grid-column: 2 / 6;
+    }
+
+    .license {
+      grid-column: 2 / 6;
+    }
+
+    .support {
+      grid-column: 6 / 10;
+
+      column-width: 160px;
+      column-gap: 20px;
+    }
+  }
+
+  @media (min-width: 1140px) {
+    body,
+    header[role="banner"],
+    footer[role="contentinfo"] {
+      grid-column-gap: 20px;
+      grid-template-columns: 1fr repeat(14, 60px) 1fr;  
+    }
+    
+    header[role="banner"],
+    footer[role="contentinfo"] {
+      grid-column: span 16;
+    }
+
+    .license {
+      grid-column: 4 / 10;
+    }
+    
+    .support {
+      grid-column: 10 / 16;
+
+      column-width: 140px;
+      column-gap: 20px;
+    }
+    
+    .content {
+      grid-column: 4 / 8;
+    }
+  }  
+
+  @media (min-width: 1300px) {
+    body,
+    header[role="banner"],
+    footer[role="contentinfo"] {
+      grid-column-gap: 20px;
+      grid-template-columns: 1fr repeat(16, 60px) 1fr;  
+    }  
+
+    .glogo {
+      grid-column: 16;
+      justify-self: flex-end;
+    }  
+
+    header[role="banner"],
+    footer[role="contentinfo"] {
+        grid-column: span 18;
+    }
+
+    .license {
+      grid-column: 2 / 11;
+    }
+
+    .support {
+      grid-column: 11 / 17;
+    }
+
+    .support li {
+      margin-left: -10px;
+      padding-left: 10px;
+      border-left: 1px solid #dcdcdc;
+    }
+
+    .content {
+      grid-column: 5 / 9;
+    }
+  }
+}
+
+@supports not (display: grid) {
```

Comment:
`@supports` may be a problem for IE?

Reply:
Hmm, yeah, I will have to deal with old IE eventually. I don't want to mess up the styles here with backwards compatibility stuff. How have you been doing it so far?

Reply:
After going through [this](https://hacks.mozilla.org/2016/08/using-feature-queries-in-css/), I think I'll just put everything above and hope grid support will undo everything correctly 🤞

Reply:
Done.

Reply:
That's pretty much how I've been doing it ^, use `supports` instead of `not supports`.